primefaces/src/main/java/org/primefaces/behavior/validate/ClientValidator.java

```diff
@@ -78,18 +78,18 @@ protected BehaviorAttribute[] getAllAttributes() {
     }
 
     public String getEvent() {
-        return eval(PropertyKeys.event, null);
```

Comment:
@tandraschko for consistency with other code and Behaviors shouldn't this technically be called `ClientValidatorBehavior`?

Reply:
yes -> new issue pls

Reply:
OK i will create one and fix it.